src/components/BodyTable/BodyTable.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  person: Person,
+  selectedSlug: string | undefined,
+};
+
+const IS_NAME = '-';
```

Comment:
What does the current name mean? We store a specific value for this constant and do not check anything.
```suggestion
const NOT_SET_VALUE = '-';
```